Migrate Cloudcraft documentation from Help Scout to Datadog (#21358)
DataDog/documentation: content/en/cloudcraft/getting-started/create-your-first-cloudcraft-diagram.md

---
title: Create your first live AWS diagram
kind: guide
---

> Cloudcraft allows you to easily import your AWS cloud environment as *live diagrams*. By reverse-engineering the architecture in your AWS account, Cloudcraft can autogenerate new diagrams or enhance existing ones, saving you hours or even days of work.
> 
> <section class="alert alert-info">
>   <p>If you would like to get a full tour of Cloudcraft and all its features, you can <a href="https://www.cloudcraft.co/request-demo" title="Book an online demo of Cloudcraft">book an online demo with a member of our team</a>.</p>
> </section>
> 

Reviewer: ```suggestion
Cloudcraft allows you to import your AWS cloud environment as *live diagrams*. By reverse-engineering the architecture in your AWS account, Cloudcraft can autogenerate new diagrams or enhance existing ones, saving you hours or even days of work.
```

> ## Your first live diagram
> 
> Start by creating a new blueprint, where we will be importing your cloud architecture.
> 
> <section class="alert alert-info">
>   <p>A blueprint contains your diagram, a budget, and all the documentation you attach to individual components and the diagram itself.</p>
> </section>
> 
> First things first, you need to connect your AWS account. Go ahead and do that by following the help article below:
> 
> - [Connect your AWS account with Cloudcraft][1]
> 
> Connected? Perfect!
> 
> Click on the **Live** tab, which is where you can choose the account you just added, scan AWS regions, generate automatic layouts, and view all resources in your AWS account.
> 
> {{< img src="cloudcraft/getting-started/create-your-first-cloudcraft-diagram/live-tab.png" alt="Screenshot of a live AWS infrastructure diagram on Cloudcraft platform with design, live, and budget tabs visible. The Live tab is highlighted." responsive="true" style="width:100%;">}}

Reviewer: ```suggestion
## Prerequisites

Before you begin, you must connect your AWS account to Cloudcraft. For more information, see [Connect your AWS account with Cloudcraft][1].

## Your first live diagram

To import your cloud architecture, you must first create a new blueprint. A blueprint contains your diagram, a budget, and all the documentation you attach to individual components and the diagram itself.

In Cloudcraft, navigate to the **Live** tab. This is where you choose your AWS account, scan AWS regions, generate automatic layouts, and view all resources in your AWS account.

{{< img src="cloudcraft/getting-started/create-your-first-cloudcraft-diagram/live-tab.png" alt="A live AWS infrastructure diagram in Cloudcraft with the Live tab highlighted." responsive="true" style="width:100%;">}}
```

> If you only added one AWS account to Cloudcraft, it will be automatically selected for you, otherwise, select the account you want from the dropdown.

Reviewer: ```suggestion
If you only added one AWS account to Cloudcraft, it's automatically selected for you, otherwise, select the account you want from the dropdown.
```

> {{< img src="cloudcraft/getting-started/create-your-first-cloudcraft-diagram/select-aws-account.png" alt="Screenshot showing the selection of an AWS account in Cloudcraft interface with an arrow pointing to a dropdown menu." responsive="true" style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="cloudcraft/getting-started/create-your-first-cloudcraft-diagram/select-aws-account.png" alt="Screenshot showing the selection of an AWS account in Cloudcraft interface with an arrow pointing to a dropdown menu." responsive="true" style="width:75%;">}}
```

Now, select the region you want to scan. You can scan and add multiple regions into a single diagram, but for now, just pick one.

> {{< img src="cloudcraft/getting-started/create-your-first-cloudcraft-diagram/select-aws-region.png" alt="Arrow pointing to the AWS region selection dropdown menu set to us-east-1 with a 'Scan now' button." responsive="true" style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="cloudcraft/getting-started/create-your-first-cloudcraft-diagram/select-aws-region.png" alt="Arrow pointing to the AWS region selection dropdown menu set to us-east-1 with a 'Scan now' button." responsive="true" style="width:75%;">}}
```

> Perfect, we are almost there.
> 
> You will notice a toggle that says **Live** or **Snapshot** just below the yellow **Scan Now** button, which tells the application which kind of diagram you want to create. If you select **Live**, the diagram will constantly update itself with information from your AWS account, while selecting **Snapshot** will create a point-in-time image, which means the diagram will never update automatically.
> 
> For this help article we will focus on the **Live** version, so go ahead and select that option. The cog icon to the right of the option allows you to further customize how your diagram will be updated, but you can ignore it for now.
> 

Reviewer: ```suggestion
Below the **Scan Now** button is a toggle that says **Live** or **Snapshot**. This tells the application which kind of diagram you want to create. If you select **Live**, the diagram continuously updates with information from your AWS account. If you select **Snapshot**, a point-in-time image is created, which means the diagram will never update automatically.

This example uses the **Live** option. Enable the toggle for **Live**. The cog icon to the right of the option allows you to further customize how your diagram is updated, but you can ignore it for now.
```
